### PR TITLE
Fix various minor issues

### PR DESCRIPTION
**Pull request by philon-msft**
These are fixes for issues I discovered while evaluating the template authoring experience in editors that support schema validation.

### CODE REVIEW

**Comment by daschult** [comment]
Looks good to me.

**Comment by OptikSage** [comment]
[X] LGTM

File: schemas/2015-08-01/Microsoft.Compute.json [hunk 1]
```diff
@@ -62,7 +67,8 @@
                     "type": "string"
                 },
                 "sku": {
-                    "type": "string"
+                    "type": "string",
+                    "default": "latest"
```
Comment (wenwu449) [inline schemas/2015-08-01/Microsoft.Compute.json]: "latest" is the default value of "version", right?

Comment (philon-msft) [inline schemas/2015-08-01/Microsoft.Compute.json]: Yes, that's the intent of this change

File: schemas/2015-08-01/Microsoft.Web.json [hunk 2]
```diff
@@ -374,7 +386,11 @@
                         }
                     }
                 }
-            }
+            },
+            "required": [
+              "name",
```
Comment (wenwu449) [inline schemas/2015-08-01/Microsoft.Web.json]: can you also add "type" and "apiVersion" as required?

Comment (philon-msft) [inline schemas/2015-08-01/Microsoft.Web.json]: There are no "type" or "apiVersion" properties on this object. Unless I misunderstand your comment?

Comment (wenwu449) [inline schemas/2015-08-01/Microsoft.Web.json]: my bad, must be lost in different objects/types. You are right.

**Pull request merged**